### ISSUE TITLE
7.0.0: Incorrect `main` value in `package.json`

### ISSUE THREAD

**Issue by srmagura** [ngneat/falso#360]
### Is this a regression?

Yes

### Description

The `package.json` in my `node_modules` for falso version 7.0.0 contains this:

```json
  "module": "./index.js",
  "main": "./index.cjs.js",
  "type": "module",
  "types": "./src/index.d.ts"
```

The value for `main` is incorrect — the file is actually named `index.cjs`. This is causing a failure when attempting to import falso from my Vitest tests.

### Please provide the exception or error you saw

```true
Error: Failed to resolve entry for package "@ngneat/falso". The package may have incorrect main/module/exports specified in its package.json.       
 ❯ packageEntryFailure ../../node_modules/vitest/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:23382:11
 ❯ resolvePackageEntry ../../node_modules/vitest/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:23379:5
 ❯ tryNodeResolve ../../node_modules/vitest/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:23113:20
 ❯ Context.resolveId ../../node_modules/vitest/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:22874:28
 ❯ Object.resolveId ../../node_modules/vitest/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:42847:46
 ❯ TransformContext.resolve ../../node_modules/vitest/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:42575:23
 ❯ normalizeUrl ../../node_modules/vitest/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:40500:34
 ❯ async file:/C:/Projects/work/fullcontext/node_modules/vitest/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:40651:47
```

### Please provide the environment you discovered this bug in

```true
Windows 11
Node 18.16.0
Vitest 0.31.3
Vite 4.3.9
```

### Do you want to create a pull request?

No

**Comment by shaharkazaz** [comment]
Thanks, I'll take a look

**Comment by shaharkazaz** [comment]
Fixed in 7.0.1
